### PR TITLE
Try chromium61

### PR DESCRIPTION
**Pull request by tejohnso**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "author": "Tyler Johnson <mail@tyler-johnson.ca> (http://github.com/tejohnso)",
   "license": "ISC",
   "devDependencies": {
-    "electron": "1.8.2",
+    "electron": "2.0.0-beta.7",
```
Comment (stulees) [inline package.json]: @tejohnso I didn't realize v2 is still in beta. Any concerns about us going stable with a beta Electron version?

**Comment by santiagonoguez** [comment]
Also, this update will need to upgrade display control build also I guess, as happened last time ?

**Comment by stulees** [comment]
@santiagonoguez Correct, DC needs to update serialport rebuild per arch/platform.

File: package.json (hunk 1 above)
Comment (tejohnso) [reply package.json]: @stulees not ideal, but it's the 7th release and it's been in beta for over 40 days now. Plus we're releasing to our own BETA to see how it performs before going to stable.

**Pull request closed without merging**